src/main/java/edu/cornell/kfs/module/ar/batch/service/impl/CuVerifyBillingFrequencyServiceImpl.java

```diff
@@ -140,6 +142,22 @@ public BillingPeriod getStartDateAndEndDateOfPreviousBillingPeriod(ContractsAndG
     public BillingPeriod getStartDateAndEndDateOfPreviousBillingPeriod(ContractsAndGrantsBillingAward award, Date calculatedLastBilledDate, AccountingPeriod currPeriod, String creationProcessTypeCode) {
         return BillingPeriod.determineBillingPeriodPriorTo(award.getAwardBeginningDate(), this.dateTimeService.getCurrentSqlDate(), calculatedLastBilledDate, ArConstants.BillingFrequencyValues.fromCode(award.getBillingFrequencyCode()), this.accountingPeriodService, creationProcessTypeCode);
     }
+    
+    /**
+     * CU Customization: The lastBilledDate should be set to the invoice creation date on the first invoice when
+     *                   Invoicing Option is Invoice By Schedule (Milestone or Predetermined) and Last Billed Date (Award) is null.
+     *                   Invoice is considered to be first invoice based on above listed criteria.
+     *                   This adjustment needs to be performed when invoices are created by BOTH Batch and Manual processing.
+     */
```

Comment:
I am not sure if we need this comment, and if we do need it, do you think it would be more helpful where this function is used in CuContractsGrantsInvoiceCreateDocumentServiceImpl

Reply:
OK, lets leave it as is